src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -30,45 +33,55 @@ public class BookController {
     private final BookService bookService;
 
     @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
     @Operation(summary = "Create a new book", description = "Create a new book")
-    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
+    public BookDto createBook(Authentication authentication,
```

Comment:
This is 'Add Category model' task. Do not add extra staff. And we do not need Authentication here